### PR TITLE
Fix eventing-rabbitmq release notes for Release Blog v1.3

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes 

- Fixes the eventing-rabbitmq release notes to use the v1.3.1 release notes. It appears as though they were using v1.2 release notes.

### CODE REVIEW

File: blog/docs/releases/announcing-knative-v1-3-release.md [hunk 1]
```diff
@@ -107,25 +107,12 @@ This helps to avoid hitting 503 errors during upgrade. ([#12617](https://github.
...
-- Improved Broker and Source README docs and Samples description and files. ([#555](https://github.com/knative-sandbox/eventing-rabbitmq/pull/555))
-
-#### 🐞 Bug Fixes
-
-- Removing the dead letter sink on a trigger will now properly fall back to the Broker's dead letter sink, if one is defined. ([#533](https://github.com/knative-sandbox/eventing-rabbitmq/pull/533))
-- Configuring messages sent into the RabbitMQ Broker to be persistent as the Queues used by the Broker are always durable.
-Now if the user set the configuration of the RabbitMQ Source Exchange and Queue to be durable, the messages are also durable. ([#560](https://github.com/knative-sandbox/eventing-rabbitmq/pull/560))
-
+- Broker URLs updated to be `http://<broker-URL>/<namespace>/<broker-name>` (#587, @gab-satchi)
+- Short testing guide for contributors, .env -> (direnv friendly).envrc (#599, @ikvmw)
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-3-release.md:115]: You'll need to format the PRs as links for them to render properly on the blog, for example

```
([#587](https://github.com/knative-sandbox/eventing-rabbitmq/pull/587))
```

**Review by psschwei — CHANGES_REQUESTED** [review]
(we shouldn't need to cherry-pick this; the blog is built from the main branch)

File: blog/docs/releases/announcing-knative-v1-3-release.md (hunk 1 above)
Comment (xtreme-sameer-vohra) [reply blog/docs/releases/announcing-knative-v1-3-release.md]: Thanks @psschwei 
Updated the PR.

**Comment by xtreme-sameer-vohra** [comment]
> (we shouldn't need to cherry-pick this; the blog is built from the main branch)

Good to know 😁 
Not sure if there is a way to cancel the request at this stage itself. I'll cancel the generated PR.

File: blog/docs/releases/announcing-knative-v1-3-release.md [hunk 2]
```diff
@@ -107,25 +107,12 @@ This helps to avoid hitting 503 errors during upgrade. ([#12617](https://github.
...
-- Improved Broker and Source README docs and Samples description and files. ([#555](https://github.com/knative-sandbox/eventing-rabbitmq/pull/555))
-
-#### 🐞 Bug Fixes
-
-- Removing the dead letter sink on a trigger will now properly fall back to the Broker's dead letter sink, if one is defined. ([#533](https://github.com/knative-sandbox/eventing-rabbitmq/pull/533))
-- Configuring messages sent into the RabbitMQ Broker to be persistent as the Queues used by the Broker are always durable.
-Now if the user set the configuration of the RabbitMQ Source Exchange and Queue to be durable, the messages are also durable. ([#560](https://github.com/knative-sandbox/eventing-rabbitmq/pull/560))
-
+- Broker URLs updated to be `http://<broker-URL>/<namespace>/<broker-name>` ([#587](https://github.com/knative-sandbox/eventing-rabbitmq/pull/587))
+- Short testing guide for contributors, .env -> (direnv friendly).envrc ([#599](https://github.com/knative-sandbox/eventing-rabbitmq/pull/599))
```
Comment (psschwei) [inline blog/docs/releases/announcing-knative-v1-3-release.md:115]:
```suggestion
- Short testing guide for contributors, converting from `.env` -> to `(direnv friendly).envrc` environment files. ([#599](https://github.com/knative-sandbox/eventing-rabbitmq/pull/599))
```

(Alternatively, maybe just drop the .env -> .envrc bit?)

**Review by psschwei — COMMENTED** [review]
tiny nits (sorry, should've noticed them on the last review)

**Comment by xtreme-sameer-vohra** [comment]
> tiny nits (sorry, should've noticed them on the last review)

Not at all !! In fact, thank you for taking the time to make up for my poor eye for detail 😁

**Comment by xtreme-sameer-vohra** [comment]
Doesn't appear that failing checks are result of the new patch changes.
/retest

**Comment by psschwei** [comment]
Yeah, looks like [Jinja released a new version recently](https://pypi.org/project/Jinja2/#history) and it's breaking the build. I think it gets pulled in by mkdocs as a dependency, but because we don't directly use it we aren't pinning it's version in [requirements.txt](https://github.com/knative/docs/blob/main/requirements.txt), though maybe we should be...

Off the top of my head, I think this can be fixed by pinning the Jinja2 version to 3.0.3 (which was the latest version up until yesterday), but need to verify that...

**Comment by psschwei** [comment]
Opened #4872 to fix the build issue

**Review by psschwei — COMMENTED** [review]
/lgtm

**Comment by xtreme-sameer-vohra** [comment]
> (we shouldn't need to cherry-pick this; the blog is built from the main branch)

Looks like the cherry pick generation failed so we're good here :)

**Pull request merged**